solution/ui/regulations/eregs-vite/src/components/subjects/PolicyResults.vue

```diff
@@ -27,33 +27,71 @@ const addSurroundingEllipses = (str) => {
     return str;
 };
 
+const getFieldVal = ({ item, fieldName }) => {
+    if (item?.resource) {
+        return item.resource[fieldName];
+    } else {
+        return item[fieldName];
+    }
+};
```

Comment:
This method is necessary (for now) because we are using two endpoints to retrieve data on the Subjects page: 
- `content-search` is there is a search query (`q`)
- `resources` / `resources/public` / `resources/internal` (depending on document type selections

These two endpoints return results in slightly different ways; `content-search` has a `resource` field that contains fields common to different document/file/link types.  The `resources` endpoint does not nest these common values in a `resource` field; they're just at the top level of the result item object.  Since we need to consume responses from both endpoints, we have to use this `getFieldVal` helper method to extract the proper value no matter which endpoint is used.